sections/security/dependencysecurity.md

```diff
@@ -0,0 +1,50 @@
+# Constantly and automatically inspect for vulnerable dependencies
+
+### One Paragraph Explainer
+
+The majority of Node.js applications rely heavily on a large number of third party modules from  NPM or Yarn package managers due to ease and speed of development. However, the downside to this benefit is the security risks of including unknown vulnerabilities into your application, which is a risk recognised by it's place in the OWASP top critical web application security risks list.
+
+There are a number of tools available to help identify third party packages in Node.js applications which have been identified as vulnerable by the community to mitigate the risk of introducing them into your project. These can be used periodically from CLI tools or included as part of your application's build process.
+
+Another thing to consider along with using these mitigation tools is to identify what dependencies are no longer being used in the application, and whether or not they are still required or could be implemented in Node.js without needing to reinvent the wheel.
+
+### Table of Contents
+- [Node Security Platform (nsp)](#node-security-platform)
+- [Snyk](#snyk)
+- [Greenkeeper](#greenkeeper)
+
+### Node Security Platform (nsp)
+Node Security Platform is a set of services targeted towards dependency management with a focus on security.
+
```

Comment:
provide an example of something highly relevant and dangerous, for example: a threat that was found in Express recently